anjali/sb-pt-exp/1.js

```diff
@@ -0,0 +1,49 @@
+// https://levelbuilder-studio.code.org/s/sb-pt-exp/lessons/1/levels/1
```

Comment:
Looks like most of this is the same validation code, so I won't leave inline comments, but lmk if there's anything we should chat about for this script!